DependencyInjection/SpClient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Italia\SpidSymfonyBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerInterface as Container;
```

Comment:
Why the name change?

Reply:
I remove "as Container", i found this in symfony examples